src/pages/Repositories.js

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import styles from "./Repositories.module.css";
+import clsx from "clsx";
+import ArrowLink from "./ArrowLink";
+
+function Repositories({reposData, reposConfig}) {
```

Comment:
Build is failing with error:

```
Docusaurus Node/SSR could not render static page with path=/Repositories because of error: Cannot read property 'map' of undefined
Docusaurus Node/SSR could not render static page with path=/Social because of error: Cannot read property 'twitter' of undefined
```

there's something wrong with passing properties to `Repositories` and `Social`, I'll have a look.

Reply:
I found [in Docusaurus docs](https://docusaurus.io/docs/creating-pages):

> All JavaScript/TypeScript files within the src/pages/ directory will have corresponding website paths generated for them. If you want to create reusable components into that directory, use the exclude option (by default, files prefixed with _, test files(.test.js) and files in __tests__ directory are not turned into pages).

so I moved all non-page components to another folder, I chose `src/components`.

Reply:
Cool, makes sense.